packages/-ember-data/tests/integration/references/record-test.js

```diff
@@ -32,75 +33,134 @@ module('integration/references/record', function (hooks) {
     assert.equal(recordReference.id(), 1);
   });
 
-  test('push(object)', function (assert) {
-    var done = assert.async();
+  test('a RecordReference can be retrieved via store.getReference(identifier) without local state', function (assert) {
+    let store = this.owner.lookup('service:store');
+    let recordReference = store.getReference({ type: 'person', id: 1 });
+
+    assert.equal(recordReference.remoteType(), 'identity');
+    assert.equal(recordReference.type, 'person');
+    assert.equal(recordReference.id(), 1);
```

Comment:
```suggestion
    assert.equal(recordReference.id(), '1');
```

note: the reason these tests pass even though the id will be coerced to string is that `equal` will type-convert. We may want to use strictEqual in the future for id tests.